package.json

```diff
@@ -99,6 +99,7 @@
     "@tsconfig/recommended": "^1.0.1",
     "@types/aws-lambda": "^8.10.106",
     "@types/bcrypt": "^5.0.0",
+    "@types/bluebird": "^3.5.38",
```

Comment:
side qn: why do we need bluebird?

Reply:
`cloudwatch.logger` uses `Bluebird.promisify`